### DISCUSSION TITLE
Thomas Paul Mann

### DISCUSSION THREAD

**Discussion by hipstersmoothie** [devtools-fm/devtools.fm#101] [Announcements]
## Background:

[Thomas' Twitter](https://twitter.com/thomaspaulmann)
[GitHub](https://github.com/thomaspaulmann)
[raycast Twitter](https://twitter.com/raycastapp)
[raycast](https://www.raycast.com/)

## Intro

Andrew: Hello, welcome to the devtools.fm podcast. This is a podcast about developer tools and the people who make them. I'm Andrew and this is my co-host Justin. 

Justin: Hey everyone! We're joined today by Thomas Paul Mann, CEO and co-founder of Raycast. Raycast is a tool for Mac that's like a supercharged spotlight. Maybe I'll leave the elevator pitch to you Thomas, ha. Before that though, would you tell our audience a little more about yourself?

## Questions

- [x] For our audience who hasn't seen it yet, how would you describe raycast?
- [x] Why did you build Raycast?
- [x] How has it changed your workflow/how you use a computer?
- [x] What's your favorite feature of Raycast?
- [x] How is Raycast itself built? What programming language did you use?
- [x] Being able to write plugins for Raycast seems super powerful? What type of experiences prompted you to build that feature?
- [x] What's the developer experience like for writing plugins?
- [x] What parts of raycast/you Mac can you build extensions for?
- [x] What are some of the coolest Raycast plugins you've seen?
- [x] How have you monetized it? What value can teams bring to a Raycast user? (It's interesting to note that some of our other guests like fig and warp are approaching monetization in similar ways). 
- [x] This spotlight like experience is very Mac-y, are there any plans to bring Raycast to other platforms?
- [x] The way we interact with computers changes very slowly, Raycast is an example of a different mode of operation. Do you think that we're on the cusp of a new way to interact with computers entirely? What do you think/wish the future looks like?

## Tooltips

Add them in the comments below!

## Outro

Andrew: Well that’s it for this week’s episode of devtools.fm. Be sure to follow us on YouTube and wherever you consume your podcasts. Thanks for listening!

**Comment by thomaspaulmann** [comment]
Could be nice to talk about our Slack Community a bit given that this is something that other companies/projects do. Few questions:

- [x] When, how and why did you decide to start a community on Slack for Raycast?
- [x] How did you grow the Community?

**Comment by thomaspaulmann** [comment]
My tooltips: I recently re-discovered newsletters for me. Here are three that I read every week:
1. [Console Newsletter - The best tools for developers](https://console.dev/)
2. [Pointer - A Reading Club For Software Developers](https://www.pointer.io/)
3. [Refind - Get smarter every day](https://refind.com/)

For tools, I rely heavily on Raycast. Few of my most used extensions:
- [Visual Studio Recent Projects](https://www.raycast.com/thomas/visual-studio-code)
- [GIF Search](https://www.raycast.com/josephschmitt/gif-search)
- [Google Translate](https://www.raycast.com/gebeto/translate)

**Comment by just-be-dev** [comment]
Tooltips:
- https://github.com/esamattis/react-zorm
- https://github.com/PipedreamHQ/pipedream

**Comment by hipstersmoothie** [comment]
My tooltips:

https://pnpm.io/
https://motion.dev
